test/integration/uri-options/uri.test.ts

```diff
@@ -53,17 +44,14 @@ describe('URI', function () {
     // in this case we are setting that node needs to be higher than 0.10.X to run
     metadata: { requires: { topology: 'single' } },
 
-    test: function (done) {
+    test: async function () {
       if (process.platform === 'win32') {
-        return done();
+        return;
       }
 
       const client = this.configuration.newClient('mongodb://%2Ftmp%2Fmongodb-27017.sock');
-
-      client.connect(function (err, client) {
-        expect(err).to.not.exist;
-        client.close(done);
-      });
+      await client.connect();
+      await client.close();
```

Comment:
I do find it a but odd to have a test that doesn't assert anything, even though it will fail if an error is thrown. Can we potentially catch on the connect and assert there is no error like before?

Reply:
Sure, we can use the standard pattern to get the exception and very that one wasn't thrown.